### PR TITLE
add errors if the live chat connection has errors or is rejected

### PR DESCRIPTION
**Pull request by kristinmerbach**

### CODE REVIEW

**Review by markgoho — CHANGES_REQUESTED** [review]
since this is 3rd party code and can't be sure of how the subscription is handled, I would recommend putting a flag on this so we can observe it on a limited basis

also, let's just log the error instead of throwing

File: app/views/shared/customer_support/_live_chat_scripts.html.erb [hunk 1]
```diff
@@ -139,4 +139,16 @@
   }
 
   const customPlugin = CXBus.registerPlugin('Custom');
+
+  <% if EnrollRegistry.feature_enabled?(:live_chat_widget_error_logging) %>
+    // throw error when the webchat gets rejected or has an error for visibility
+
+    customPlugin.subscribe('WebChat.rejected', function(e){
+      (console.error || console.log).call(console, e.stack || e);
```
Comment (markgoho) [inline app/views/shared/customer_support/_live_chat_scripts.html.erb:147]: why not just `console.error(e)`?

Comment (kristinmerbach) [reply app/views/shared/customer_support/_live_chat_scripts.html.erb:147]: muscle memory from having to support IE9 where error.stack was not a thing. I can take it out.

**Review by markgoho — COMMENTED** [review]
sorry I'm still thinking about this pattern and when to use it vs. not -- I'm wondering what value we get from the resource registry indirection vs. using an environment variable directly in the view

e.g.
``` erb
<% if EnrollRegistry.feature_enabled?(:live_chat_widget_error_logging) %>

vs

<% if ENV['LIVECHAT_ERROR_LOGGING_IS_ENABLED'] %>
```

**Review by markgoho — APPROVED** [review]

**Pull request merged**